### DISCUSSION TITLE
Multi select label show only text

### DISCUSSION THREAD

**Discussion by krngd2** [JedWatson/react-select#5210] [Q&A] [answered]
Hello, 

I am trying to show the number of items selected in the label.
There seems to be no way as components MultiValue, MultiValueContainer, MultiValueLabel all are looping for each value. I just want to show some text. How can I do it ?

**Comment by Rall3n** [comment, marked as answer]
Hello @krngd2,
this has already been asked multiple times. Here are two issues with available solutions: https://github.com/JedWatson/react-select/issues/3463#issuecomment-469875204, https://github.com/JedWatson/react-select/issues/2790#issuecomment-460636017.

Additionally you should consider [this comment](https://github.com/JedWatson/react-select/issues/2597#issuecomment-613871200) for these solutions.
